bofire/domain/constraints.py

```diff
@@ -213,9 +195,14 @@ def as_greater_equal(self) -> Tuple[List[str], List[float], float]:
         """
         return self.features, [-1.0 * c for c in self.coefficients], -1.0 * self.rhs
 
+    # TODO: from_greater_equal should take the object as input
     @classmethod
     def from_greater_equal(
-        cls, features: List[str], coefficients: List[float], rhs: float
+        cls,
+        features: List[str],
+        coefficients: List[float],
+        rhs: float,
+        **kwargs,
```

Comment:
why do we need the kwargs here?

Reply:
When calling this method, we pass **constraint.dict() as arguments.
In addition to the fields desired here, the newly introduced `type` property is also passed along.
Of course, this leads to errors.

This was the easier solution than removing `type` from all inputs.
IMO this was a quick fix, because I would rather go for:
- pass the actual models as arguments instead of their properties, or
- change the calls to pass `x=model.x` instead of `**`

Reply:
i will change it to the second option